src/psyclone/tests/test_files/gocean1p0/new_iteration_space.psyclone

```diff
@@ -14,6 +14,38 @@ iteration-spaces=go_offset_sw:go_ct:internal_we_halo:1:2:3:4
                  go_offset_sw:go_ct:n_halo:{start}:{start}:{start}:{stop}+1
                  go_offset_sw:go_ct:north_east_corner:{start}-2:{start}-1:{stop}:{stop}+1
 
+grid-properties = go_grid_xstop: {0}%%grid%%subdomain%%internal%%xstop: scalar,
```

Comment:
Please could you add a copyright/licence notice to this file.

Reply:
Done.